index.bs

```diff
@@ -742,6 +742,48 @@ self.onmessage = async function(e) {
 }
 </pre>
 
+# Implementation advice # {#implementation-advice}
+
+This section is informative.
+
+## Use with multiple consumers
+
+There are use cases where the programmer may desire that a single stream of frames
+is consumed by multiple consumers. However, the standard tee() operation is problematic
+in this context:
+
+*   It defeats the backpressure mechanism that guards against excessive queueing
+*   It creates multiple links to the same buffers, meaning that the question of which
+    consumer gets to destroy() the buffer is a difficult one to address
+
+Therefore, the use of tee() with Streams containing media should only be done when
```

Comment:
I would start this section by noting that:

Using two (or more) MSTProcessors on the same track gives you two streams that can be used independently by two consumers, and this is in general the best way to proceed for multiple consumers.

The considerations below are for the case when, for some reason, the application wants two consumers for a single stream from a single MSTProcessor (I don't think a use case for this has been identified).
In that case, tee() is problematic and the advice given here makes sense.
However, in most cases, using two (or more) processors should be a simpler and better solution.

Reply:
A typical (imagined) use case is where you have a funny-hat-applier pipeline and wish to display the result in a local video window (using MSTG and a <video> tag) and passing the frames to a WebCodec encoder.

Reply:
This looks like a valid use case. Let's mention it at the start of this section
and also add a snippet in the examples section (the snippet can be a follow-up CL).

Reply:
It would also be good to note that clone() is a lightweight operation. The new VideoFrame is just a new reference to the same underlying media data as in the original VideoFrame.

Reply:
According to Bernard it's somewhat unclear how the underlying buffer is handled (whether it's invalidated, copied or refcounted)...... we'll see.. New version coming.

Reply:
Clone is 100% a refcount. From the spec:
clone() Creates a new VideoFrame with a reference to the same media resource.